getting-started/jdbc/docker-compose.yml

```diff
@@ -28,6 +28,7 @@ services:
       - "8182:8182"
       # Optional, allows attaching a debugger to the Polaris JVM
       - "5005:5005"
+    shm_size: 1gb
```

Comment:
Is this required?

Reply:
I found that the benchmark mentioned in the description very consistently fails with the default memory footprint of 128MB -- it strikes me that this is a very small heap for Polaris. The entity cache along is capped around 100MB. So while it's not required, I think it's a good idea.